### PR TITLE
Added link to number all motions of a category (work in progress)

### PR DESCRIPTION
**Pull request by emanuelschuetze**
contains commit of #2225

### CODE REVIEW

File: openslides/motions/static/js/motions/site.js [hunk 1]
```diff
@@ -1008,6 +1009,19 @@ angular.module('OpenSlidesApp.motions.site', ['OpenSlidesApp.motions'])
         $scope.delete = function (category) {
             Category.destroy(category.id);
         };
+
+        // number all motions in selected category
+        $scope.numbering = function (category) {
+            $http.patch('/rest/motions/category/' + category.id + '/numbering/', {})
```
Comment (emanuelschuetze) [inline openslides/motions/static/js/motions/site.js]: With this patch request I get a 405
`HTTP PATCH /rest/motions/category/3/numbering/ 405 `
What's the right syntax to start numbering? Please add these syntax to docstring.

**Comment by normanjaeckel** [comment]
I changed to POST and tested. It should work now. Please rebase your commit on top of this (rebased) branch.

**Comment by normanjaeckel** [comment]
This pull request is outdated.

**Comment by emanuelschuetze** [comment]
Ok, works now with POST request.

**Pull request closed without merging**